specification/columns/regionname.md

```diff
@@ -0,0 +1,31 @@
+# Region Name
+
+Region Name is a provider assigned name for an isolated geographic area where a [*resource*](#glossary:resource) is provisioned in or a [*service*](#glossary:service) is provided from. Region Name is commonly used for scenarios like analyzing cost and unit prices based on where *resources* are deployed. 
+
+Region Name MUST be present in the billing data and MUST be of type String. Region Name MUST NOT be null when a *resource* or *service* is operated in or managed from a distinct region by the Provider. Region Name values MUST be consistent within the provider and MUST be the same values used to indicate the region when provisioning or purchasing the *resource* or *service*. The value of the dimension Region Name SHOULD be a proper noun.
```

Comment:
Similar as in https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/352, we need to say something like:

```
The Region Name column MUST be present in the billing data when the provider supports deploying resources or services within an *region*.
```

Udam has a few examples here: https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/352